backends/plugins/sdl/sdl-provider.cpp

```diff
@@ -29,6 +29,10 @@
 
 #include "backends/platform/sdl/sdl-sys.h"
 
+#ifdef MACOSX
+#include "backends/platform/sdl/macosx/macosx_wrapper.h"
+#endif
+
```

Comment:
Thank you for these suggestions.

I pushed a new commit trying to implement that. Does it look good to you? Thanks.

Reply:
The implementation looks fine, but I would suggest to have the files in `backends/plugins/` (for example in `backends/plugins/sdl/macosx/`) where all the other plugins providers are rather than in `backends/platform/`.

Reply:
Ah, I was hesitating, but it looks like I picked the wrong one 😁

I've changed that, squashed a bit and also applied your commit message suggestions.